sunbird_service_api_test/src/test/java/org/sunbird/integration/test/page/CreatePageTest.java

```diff
@@ -0,0 +1,132 @@
+package org.sunbird.integration.test.page;
+
+import com.consol.citrus.annotations.CitrusTest;
+import com.consol.citrus.testng.CitrusParameters;
+import javax.ws.rs.core.MediaType;
+import org.springframework.http.HttpStatus;
+import org.sunbird.common.action.OrgUtil;
+import org.sunbird.common.action.PageUtil;
+import org.sunbird.integration.test.common.BaseCitrusTestRunner;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CreatePageTest extends BaseCitrusTestRunner {
+
+  public static final String PAGE_TEST_NAME_CREATE_ROOT_ORG_SUCCESS = "testCreateRootOrgSuccess";
```

Comment:
Rename PAGE_TEST_NAME_CREATE_ROOT_ORG_SUCCESS to BT_TEST_NAME_CREATE_ROOT_ORG_SUCCESS.